src/DynamoCore/DSEngine/CodeCompletionServices.cs

```diff
@@ -19,10 +20,8 @@ public class CodeCompletionServices
         public static string[] KeywordList = {Keyword.Def, 
                                         Keyword.If, Keyword.Elseif, Keyword.Else, 
                                         Keyword.While, Keyword.For, Keyword.In, Keyword.Continue,  
-                                        Keyword.Int, Keyword.Double, Keyword.String, Keyword.Bool, Keyword.Char, 
```

Comment:
whats theimplication of removing: Keyword.Int, Keyword.Double, Keyword.String, Keyword.Bool, Keyword.Char
?